product_specifications/12_xfomv_e/detail.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Deactivation of repository
+
+        Background:
+            Given that a Consortium Lead or Member cannot find a suitable host
+
+        Scenario: Fabrica de-activation
+            Given that a staff_admin user is in the Consortium Lead or Member
+             When the user select in-active
+             Then the member and all its repositories are to be labeled in-active in both Fabrica and the REST API
+              And all the DOIs of the member should be set to 'registered'
+
+Feature: Thombstone page for DOIs with broken links of inactive repositories
+
+        Background:
+            Given that a Repository has been set inactive
+              And that the DOI was set to 'registered' state by inactive workflow
+        
+        Scenario: set tombstone page 
+             When a user land on the inactive repository page
+             Then show link checker statistics for the repository using a pie graph
+             When a user clicks on the pie graph
+             Then the user is directed to a page where they are able to filter DOIs by error code and bulk select DOIs to inactivate
+
+        Scenario: landing on tombstone page
+             When an inactive DOI from the inactive repository is resolved to its thombstone page
+             Then the resolution should lead the user to a page in Commons (last metadata)
+              And DOI Commons page should include a [Non-Maintained] Label
```

Comment:
One Problem that I foreseen with using Commons to show tombstone pages is that Commons only shows "findable" DOIs. All the DOIs of an inactive repository are by default "registered" that would, I think, make complicated to show stuff in Commons. Plus Commons hooks up many features that are not available to "registered"-only DOIs, such as content-negotiation, orcid-claiming.

The alternative I was proposing is to use Fabrica. We can set up tombstone pages that need to permission for "registered"-only DOIs. We have done something similar with page for the research profile (see https://doi.datacite.org/users/0000-0003-3484-6875)

Reply:
But if we point tombstone pages to Fabrica we won't be able to show the existing links and citations etc.

Reply:
yeah that feature wouldn't not be shown unless we added to fabrica.